src/Tizen.NUI.Components/Theme/DefaultTheme.cs

```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
 
                 // Picker
                 (new ExternalThemeKeyList(typeof(Picker), typeof(PickerStyle)))
+                    .Add<Size>("/Size", (ViewStyle style, Size value) => ((ViewStyle)style).Size = value)
+                    .AddSelector<float?>("/ItemTextLabel/PixelSize", SetButtonTextPixelSize, ControlState.Selected)
+                    .Add<Size>("/ItemTextLabel/Size", (ViewStyle style, Size value) => ((ButtonStyle)style).Text.Size = value)
```

Comment:
Please change this as follows.

.AddSelector<Color>("/ItemTextLabel/TextColor", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.TextColor = value, ControlState.Selected)

Reply:
Done